doc/rst/index.rst

```diff
@@ -1,22 +1,49 @@
 oneAPI Deep Neural Network Library (oneDNN) Developer Guide and Reference
 =========================================================================
 
+oneAPI Deep Neural Network Library (oneDNN) is an open-source cross-platform performance library of basic building blocks for deep learning applications.
+
+The library is optimized for Intel(R) Architecture Processors, Intel Graphics, and Arm(R) 64-bit Architecture (AArch64)-based processors. oneDNN has experimental support for the following architectures: NVIDIA* GPU, AMD* GPU, OpenPOWER* Power ISA (PPC64), IBMz* (s390x), and RISC-V.
+
+oneDNN is intended for deep learning applications and framework developers interested in improving application performance. Deep learning practitioners should use one of the applications enabled with oneDNN.
+
+
 .. toctree::
+   :caption: About
+   :hidden:
    :maxdepth: 1
 
-   build_and_link
-   programming_model
+   Introduction<self>
+
+.. toctree::
+   :caption: Get Started 
+   :maxdepth: 1
+
+   system_requirements
+   dev_guide_build
+   dev_guide_build_options
+   dev_guide_link
+   dev_guide
```

Comment:
`dev_guide` does not reflect contents of this section and we'll use this name in other places. Maybe 'basics' or 'intro'?

Reply:
That's one option. Conceptually this is an intro to oneDNN API, so something along these lines would make sense. Also note, that these pages names are prefixed with `dev_guide` for consistency.